src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOTalon.kt

```diff
@@ -359,7 +358,7 @@ class SwerveModuleIOTalon(
     steeringPositionQueue.clear()
   }
 
-  override fun configureDrivePID(
+  override fun configDrivePID(
```

Comment:
why did we shorten all the names btw? nitpick, but imo it makes some things harder to read. you don't have to change it cuz it's already changed, but just be cognizant of it in the future bc imo, more descriptive names are easier to understand at first glance.

Reply:
changed back 👍